docs/components/constants.js

```diff
@@ -0,0 +1,59 @@
+import * as styledSystem from 'styled-system'
```

Comment:
Having this file here is temporary until we setup yarn workspaces.

Reply:
^ We should probably write a code comment that mentions that this file is temporary.